mkdocs.yml

```
@@ -9,9 +9,50 @@ repo_url: https://github.com/logicalclocks/feature-store-api
 edit_uri: ""
 
 nav:
-  - Introduction: index.md
-  - Quickstart: quickstart.md
-  - Hopsworks.ai:
+  - Feature store: 
```

can you capitalize the s, please :)
```suggestion
  - Feature Store: 
```